### ISSUE TITLE
Reporting

### ISSUE THREAD

**Issue by bohtho** [hobbyquaker/node-red-contrib-zigbee#79]
I really like your implementation bringing zigbee-herdsman into Node-RED.

I’ve previously used other zigbee controller software and am just wondering about only having a checkbox for reporting per device.

Should the max min intervals and reportable change be configurable?

And am I right in that the reporting intervals and reportable change is not pulled from the per device settings in zigbee-herdsman-converters?

**Comment by bohtho** [comment]
Also, what does the "Config" option do (which is available on only one of my devices in the devices list in the UI) ?

**Comment by svefro** [comment]
Did you figure out what the checkboxes do? i have config on 2 devices and ping available on 1 device.

**Comment by bohtho** [comment]
Nope. And I don’t really see any immediate difference on the devices where I have those options.
